qa/evil-tests/src/test/java/org/opensearch/bootstrap/SystemCallFilterTests.java

```diff
@@ -39,7 +39,7 @@
 public class SystemCallFilterTests extends OpenSearchTestCase {
 
     /** command to try to run in tests */
-    static final String EXECUTABLE = Constants.WINDOWS ? "calc" : "ls";
+    static final String[] EXECUTABLE = new String[] { Constants.WINDOWS ? "calc" : "ls" };
```

Comment:
The single string version of `Runtime#exec` is deprecated in favor of the version that takes a string array. We just pass a single executable command so that's why changing this to a single-element array fixes the deprecation warning.